docs/dev/api/rdc.md

```diff
@@ -1322,3 +1331,387 @@ No payload is returned with the successful deletion.
+
+<table id="table-api">
+  <tbody>
+    <tr>
+     <td><code>device_id</code></td>
+     <td><p><small>| PATH | REQUIRED | STRING |</small></p><p>The unique identifier of a device in the Sauce Labs data center. You can look up device IDs using the <a href="#get-devices">Get Devices</a> endpoint.</p></td>
+    </tr>
+  </tbody>
+</table>
+
+<Tabs
+groupId="dc-url"
+defaultValue="us"
+values={[
+{label: 'United States', value: 'us'},
+{label: 'Europe', value: 'eu'},
+]}>
+
+<TabItem value="us">
+
+```jsx title="Sample Request"
+curl -u "$SAUCE_USERNAME:$SAUCE_ACCESS_KEY" --location \
+--request DELETE 'https://api.us-west-1.saucelabs.com/v1/rdc/device-management/devices/iPad_Pro_11_14_2018_real/team' | json_pp
+```
+
+</TabItem>
+
+<TabItem value="eu">
+
+```jsx title="Sample Request"
+curl -u "$SAUCE_USERNAME:$SAUCE_ACCESS_KEY" --location \
+--request DELETE 'https://api.eu-central-1.saucelabs.com/v1/rdc/device-management/devices/iPad_Pro_11_14_2018_real/team' | json_pp
+```
+
+</TabItem>
+</Tabs>
+
+#### Responses
+
+<table id="table-api">
+<tbody>
+  <tr>
+    <td><code>200</code></td>
+    <td colSpan='2'>Device successfully removed from team.</td>
+  </tr>
+</tbody>
+<tbody>
+  <tr>
+    <td><code>404</code></td>
+    <td colSpan='2'>Device not found.</td>
+  </tr>
+</tbody>
+</table>
+
+No payload is returned with the successful removal.
+
+</details>
+
+---
+
+### Update Device Settings
+<details>
+<summary><span className="api put">PUT</span> <code>/v1/rdc/device-management/devices/&#123;device_id&#125;/settings</code></summary>
+<p/>
+
+Update device settings to allow apps, system apps, and accounts to persist between sessions. Check out the 
+[available device settings](/basics/acct-team-mgmt/private-device-mgmt/#app-allow-list) for more information on each setting.
+
+#### Parameters
+
+<table id="table-api">
+  <tbody>
+    <tr>
+     <td><code>device_id</code></td>
+     <td><p><small>| PATH | REQUIRED | STRING |</small></p><p>The unique identifier of a device in the Sauce Labs data center. You can look up device IDs using the <a href="#get-devices">Get Devices</a> endpoint.</p></td>
+    </tr>
+  </tbody>
+  <tbody>
+    <tr>
+     <td><code>appWhitelist</code></td>
+       <td><p><small>| BODY | REQUIRED | ARRAY |</small></p><p>Persist installed apps and app data between sessions.</p></td>
+    </tr>
+  </tbody>
+  <tbody>
+    <tr>
+     <td><code>accountWhitelist</code></td>
+       <td><p><small>| BODY | REQUIRED | ARRAY |</small></p><p>Preserve store and payment account sign-ins between sessions for each account email.</p></td>
+    </tr>
+  </tbody>
+  <tbody>
+    <tr>
+     <td><code>systemAppAllowlist</code></td>
+       <td><p><small>| BODY | REQUIRED | ARRAY |</small></p><p>Access iOS preinstalled applications.</p></td>
+    </tr>
+  </tbody>
+  <tbody>
+    <tr>
+     <td><code>applePaySupportEnabled</code></td>
+       <td><p><small>| BODY | REQUIRED | BOOLEAN |</small></p><p>Enable to test Apple Pay.</p></td>
```

Comment:
@yuyi-sl customer needs to reach out to CSM and customer support after this. can you mentioned, after enabling this flag, you need to reach out to your accounts/support team to configure the device additionally

Reply:
added additional configuration required and link to documentation